src/table/body-cell/index.tsx

```diff
@@ -85,14 +79,8 @@ function TableCellEditable<ItemType>({
     <TableTdElement
       {...rest}
       nativeAttributes={tdNativeAttributes as TableTdElementProps['nativeAttributes']}
-      className={clsx(
-        className,
-        styles['body-cell-editable'],
```

Comment:
This style is now added in the TableTdElement: `isEditable && styles['body-cell-editable']`